### PR TITLE
SLA update

### PR DESCRIPTION
**Pull request by karl-johan-grahn**
Cost and support update

### CODE REVIEW

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-71-18af3e38`

File: docs/content/openshift-as-a-service/technical-support.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Service Type | Non-Prod | Prod
 --- | --- | ---
 Incident response time | Up to 30 min | Up to 15 min
 24x7x365 monitoring and handling cluster incidents | No | Yes
-Technical advice on cluster architecture and applications during regular business hours | Yes | Yes
+Technical advice on cluster architecture and applications during regular business hours | Yes, billed separately | Yes, billed separately
 Voice consultations with lead engineers during regular business hours | No | Yes
```
Comment (rasheedamir) [inline docs/content/openshift-as-a-service/technical-support.md]: @karl-johan-grahn this should be billed separately as well

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-71-a25962f0`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.46`

**Pull request merged**